src/content/getting-started/install-an-application/_index.md

```diff
@@ -81,22 +75,22 @@ __Note__: You can [create your own catalog]({{< relref "/vintage/getting-started
 
 To browse which applications are available in the catalog you can run the following command:
 
-```nohighlight
+```text
```

Comment:
It's the other way around – pure commands should be highlighted with `sh`, command line + comand output should be `text` (or `nohighlight` which should be the same).

Reply:
ups sorry I mixed them up, yeah is what I wanted

Reply:
fixed